### PR TITLE
Fix: DO-2441 tiers confg has a typo on dara and add layering algo prop

### PR DESCRIPTION
**Pull request by patricia-causalens**
## Motivation and Context

Exposing layering_algorithm on dara side and fixinf a typo where TiersConfg was missing the i

## Implementation Description

Replaced TiersConfg name to TiersConfig
Updated ui deps to 1.5.2
Added layering_algorithm to the builder for PlanarLayout

## Any new dependencies Introduced

dara-ui updated to 1.5.2

## How Has This Been Tested?

Locally on a demo app

## PR Checklist:

- [x] I have implemented all requirements? (see JIRA, project documentation).
- [x] I am not affecting someone else's work, If I am, they are included as a reviewer.
- [ ] I have added relevant tests (unit, integration or regression).
- [x] I have added comments to all the bits that are hard to follow.
- [x] I have added/updated Documentation.
- [x] I have updated the appropriate changelog with a line for my changes.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (krzysztof-causalens) [inline pnpm-lock.yaml:1]: Why is the diff here so large? Just curious, can't view the diff on GitHub because of the size 😅

**Review by krzysztof-causalens — APPROVED** [review]

File: pnpm-lock.yaml
Comment (krzysztof-causalens) [reply pnpm-lock.yaml:1]: 🤔 you shouldn't be running the lockfile through the formatter, can you include the original generated by pnpm?

Comment (patricia-causalens) [reply pnpm-lock.yaml:1]: Sorry that was the originally created, I didn't run anything one it, it was created by the update-ui-deps command. I think the formatter is a separate issue I am having.

Comment (krzysztof-causalens) [reply pnpm-lock.yaml:1]: Right, I see. Okay let me have a look at the diff locally just as a sanity check nothing weird happened, it is unusually large

Comment (krzysztof-causalens) [reply pnpm-lock.yaml:1]: It seems like the previous lockfile version on master was autoformatted in #177 so this is just undoing it, this is fine :)

**Pull request merged**